torch/utils/collect_env.py

```diff
@@ -37,6 +38,10 @@
     'nvidia_driver_version',
     'nvidia_gpu_models',
     'cudnn_version',
+    'is_xpu_available',
```

Comment:
I think `is_xpu_available` is not a must. We can infer this information through `torch.version.xpu` and the device name.